build.sbt

```
@@ -152,10 +153,19 @@ lazy val tamer = project
     Test / console / scalacOptions := (Compile / console / scalacOptions).value
   )
 
+lazy val doobie = project
+  .in(file("doobie"))
+  .dependsOn(tamer)
+  .settings(commonSettings)
+  .settings(
+    name := "doobie",
```

Yeah, we probably want to prefix all modules' artifacts with `tamer-` now that we no longer have only one. This means
```suggestion
    name := "tamer-doobie",
```
And likely `tamer-core` the other one.

---

As opposed to just just `tamer`, because the module in itself doesn't do much without a supporting module to fetch data. Done.